wafer/talks/models.py

```diff
@@ -268,3 +274,51 @@ class TalkUrl(models.Model):
     description = models.CharField(max_length=256)
     url = models.URLField()
     talk = models.ForeignKey(Talk, on_delete=models.CASCADE)
+
+
+@python_2_unicode_compatible
+class Review(models.Model):
+    talk = models.ForeignKey(Talk, on_delete=models.CASCADE,
+                             related_name='reviews')
+    reviewer = models.ForeignKey(settings.AUTH_USER_MODEL,
+                                 on_delete=models.CASCADE)
+
+    notes = models.TextField(
+        null=True, blank=True,
+        help_text=_("Comments on the proposal"))
+
+    def __str__(self):
+        return u'Review of %s by %s (%i)' % (
+            self.reviewer, self.talk.title, self.total_score)
+
+    @property
+    def total_score(self):
+        return self.scores.aggregate(total=models.Sum('value'))['total']
+
+
+@python_2_unicode_compatible
+class ReviewAspect(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
```

Comment:
Perhaps we should add some sort of review weighting here?

I can imagine lots of cases people would want to place more emphasis on a particular aspect during the review process.

Reply:
Yeah, it's hard to imagine any kind of pluggable score aggregation that's going to be clean.

Weights could make sense.

Reply:
```
17:48 < gwolf> (coming up with the right set of aspects and the right weighing seems like too byzantine ;-) )
```